docs/release/trg-0/trg-1-08.md

```diff
@@ -0,0 +1,31 @@
+---
+title: TRG 1.08 - Interfaces Documentation
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status     | Created      | Post-History          |
+|------------|--------------|-----------------------|
+| Draft      | 31-Jan-2024  | created initial Draft |
+
+## Why
+
+Interfaces Documentation is crucial for integration testing and ensuring seamless interoperability for Developers consuming the product API
+
+## Description
+
+- The interface documentation **has to** be stored in the folder: ```docs/interfaces/```
+- API documentation contains all relevant interfaces for integration testing and is completed for the relevant product version.
```

Comment:
I would say __all__ interfaces are required. Not only relevant. I think it would be too hard do distinguish relevant and non-relevant